client/src/components/DataFiles/DataFilesSystemSelector/DataFilesSystemSelector.js

```diff
@@ -43,15 +77,22 @@ const DataFilesSystemSelector = ({ systemId, section, disabled }) => {
             {system.name}
           </option>
         ))}
+        <option value="shared">Shared Workspaces</option>
```

Comment:
Please account for this PR, now in 213.

https://github.com/TACC/Frontera-Portal/pull/278